apps/ejabberd/src/mod_push_service_mongoosepush.erl

```diff
@@ -123,6 +123,44 @@ http_notification(Host, Method, URL, ReqHeaders, Payload) ->
 %% Helper functions
 %%--------------------------------------------------------------------
 
+%% Create notification for API v1
+make_notification(v1, _Notification, #{<<"silent">> := <<"true">>}) ->
```

Comment:
I appreciate you care about backward compatibility. Push notifications were added to MongooseIM in 2.1.0beta1 as the name suggest there may be significant changes before the final 2.1.0. IMO, it's better to remove support for the old version now. As I understand we are going to use the new API anyway.